config/locales/en.yml

```diff
@@ -69,6 +69,9 @@ en:
   ticket_number: "Ticket Number"
   assigned_to_label: "Assigned to"
   created_on: "Created on"
+  ticket_secret_url_notice: |
+    You are viewing this ticket using the secret link %{url}.
+    Anyone with this link can view and reply to this ticket.
 
```

Comment:
new keys to en.yml should be copied to all language files as comments to mark untranslated keys